example_project/example_project/urls.py

```diff
@@ -21,6 +21,9 @@
     url(r'^orchestra/accounts/',
         include('registration.backends.default.urls')),
 
+    # Optionally include these routes to enable user hijack functionality.
+    url(r'^orchestra/switch/', include('hijack.urls')),
```

Comment:
please update docs, as this changes [install instructions](http://orchestra.readthedocs.org/en/stable/getting_started.html#install-and-configure-orchestra)